src/net.cpp

```diff
@@ -1403,16 +1406,10 @@ static void ThreadMapPort()
     struct UPNPDev * devlist = nullptr;
     char lanaddr[64];
 
-#ifndef UPNPDISCOVER_SUCCESS
-    /* miniupnpc 1.5 */
-    devlist = upnpDiscover(2000, multicastif, minissdpdpath, 0);
-#elif MINIUPNPC_API_VERSION < 14
-    /* miniupnpc 1.6 */
```

Comment:
Why remove the version comments?

Reply:
They are irrelevant now. See: [miniupnpc/apiversions.txt](https://github.com/miniupnp/miniupnp/blob/master/miniupnpc/apiversions.txt)

Reply:
I don't see how they are irrelevant...

Reply:
There was not `MINIUPNPC_API_VERSION` macro before version 1.7. Therefore, different ways were used to distinguish between different versions. Comments were needed to make those "hacks" obvious.
Nowadays, the difference in the `upnpDiscover()` function signature is clear stated in different API versions.

Particularly,
```
API version 14
miniupnpc.h
  add ttl argument to upnpDiscover() upnpDiscoverAll() upnpDiscoverDevice()
...
  updated macro :
    #define MINIUPNPC_API_VERSION 14
```